algorithms/HPdutility/inst/tests/test_hpdsample.R

```diff
@@ -0,0 +1,180 @@
+
+library(distributedR)
+library(HPdutility)
+distributedR_start()
```

Comment:
As a convention, we call neither "library(distributedR)" nor "distributedR_start()" inside test files.